src/components/TextLink/styledComponent.js

```diff
@@ -0,0 +1,6 @@
+import styled from 'styled-components/native';
+import colors from '../../constants/colors'
```

Comment:
Codacy has a fix for the issue: [Insert `;`](https://app.codacy.com/gh/BC-MAY-21-RN/proyecto-proy-t01/pullRequest?prid=7872340)
```suggestion
import colors from '../../constants/colors';
```